index.js

```diff
@@ -86,6 +86,20 @@ function createBot (options = {}) {
   return bot
 }
 
+function compareVersions (testedVersion, currentVersion) {
+  testedVersion = testedVersion.split('.')
+  currentVersion = currentVersion.split('.')
+
+  if (testedVersion.length === 2) testedVersion.push(0)
```

Comment:
Note the compare function here won’t work with snapshots, or pre-release versions and might break in the future with hyphen releases or other version strings that might not conform with int.int.int.

Reply:
Ah, that's true

Reply:
this should be fixed
you can use the protocol version from mcdata to make a range comparison version checking function